_integration-schemas/3plcentral/locations.md

```diff
@@ -0,0 +1,139 @@
+---
+tap: "3plcentral"
+version: "1.0"
+
+name: "locations"
+doc-link: "http://api.3plcentral.com/rels/"
+singer-schema: "https://github.com/singer-io/tap-3plcentral/blob/master/tap_3plcentral/schemas/locations.json"
+description: "This table contains information about locations."
+
+replication-method: "Full Table"
+
+api-method:
+    name: "3PL Central REL documentaion"
+    doc-link: "http://api.3plcentral.com/rels/inventory/locations"
+
+attributes:
+  - name: "location_id"
+    type: "integer"
+    primary-key: true
+    description: ""
+    foreign-key-id: "location-id"
+
+  - name: "facility_id"
+    type: "integer"
+    primary-key: true
+    description: ""
+    foreign-key-id: "facility-id"
+      
+  - name: "customer_identifier"
+    type: "object"
+    description: ""
+    subattributes:
+      - name: "external_id"
+        type: "string"
+        description: ""
+      - name: "id"
+        type: "integer"
+        description: ""
+        foreign-key-id: "customer-id"
+      - name: "name"
+        type: "string"
+        description: ""
+
+  - name: "description"
+    type: "string"
+    description: ""
+  
+  - name: "item_traits"
+    type: "object"
+    description: ""
+    subattributes:
+      - name: "expiration_date"
+        type: "date-time"
+        description: ""
+      - name: "item_identifier"
+        type: "object"
+        description: ""
+        subattributes:
+          - name: "id"
+            type: "integer"
+            description: ""
+            foreign-key-id: "item-id"
```

Comment:
I think I saw this as `item-identifier` somewhere else - I'd double-check if this and that other attribute are supposed to tie together.